src/status_im2/contexts/chat/messages/list/view.cljs

```diff
@@ -223,7 +224,7 @@
              :display-name    display-name
              :online?         online?
              :profile-picture photo-path}])]
-        [rn/view {:style style/name-container}
+        [rn/view
```

Comment:
Wondering if we need the outer view at all.

Reply:
removed